MinecraftClient/McClient.cs

```diff
@@ -2125,6 +2125,550 @@ public bool DoWindowAction(int windowId, int slotId, WindowActionType action)
                             }
                         }
                         break;
+                    case WindowActionType.ShiftRightClick: //added shiftrightclick action here it's same with shiftclick
```

Comment:
If the code here is the same as `ShiftClick`, you don't need to copy the whole switch case. Instead, do it like this:
```cs
case WindowActionType.ShiftClick:
case WindowActionType.ShiftRightClick:
    // The switch case will execute for both ShiftClick and ShiftRightClick
```

Reply:
oh i didn't know that thanks.

Reply:
done it